apps/dashboard/src/@/components/ui/ScrollShadow/ScrollShadow.tsx

```diff
@@ -71,8 +71,15 @@ export function ScrollShadow(props: {
       });
     });
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +68 to +72]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->why is this double nested btw?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/4698?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
because browsers are weird - https://youtu.be/cCOL7MC4Pl0?si=QdvmmLN4vZPdlcPt&t=1274 

( watch at 21:14 )

Reply:
makes total sense to me with that explanation :)